IrConsoleApp/Lecture4/Lecture4Homework.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IrynaZharnasek.Lecture4
+{
+    public class Lecture4Homework
+    {
+        public static void Task1()
```

Comment:
I think this task works a bit incorrectly. So, how it's expected to work: as a user I input 4 number and I expected that the program will count result as 1+2+3 and show me the common result = 6
now it works like this: 
![image](https://user-images.githubusercontent.com/55654931/201092814-ddfa072d-8fb7-41bd-ab50-c20e6898dc5d.png)

Reply:
Done